srcpkgs/SDRPlusPlus/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'SDRPlusPlus'
+pkgname=SDRPlusPlus
+version=1.0.4
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DOPT_BUILD_AIRSPYHF_SOURCE=OFF
+ -DOPT_BUILD_PLUTOSDR_SOURCE=OFF"
+hostmakedepends="cmake pkg-config"
+makedepends="glfw-devel librtlsdr-devel glew-devel libhackrf-devel libusb-devel
+ SoapySDR-devel libairspy-devel rtaudio-devel alsa-lib-devel pulseaudio-devel
+ jack-devel fftw-devel volk-devel"
+short_desc="SDR++ is a cross-platform and open source SDR software"
```

Comment:
this is a bit of a useless description, and should generally be in the form of a phrase

Reply:
i guess then something like "open source SDR software"

Reply:
> i guess then something like "open source SDR software"

Updated